packages/shared/src/lib/tables/cell.ts

```diff
@@ -0,0 +1,11 @@
+import { Static, Type } from '@sinclair/typebox'
+import { BaseModelSchema } from '../common'
+
+export const Cell = Type.Object({
+    ...BaseModelSchema,
+    recordId: Type.String(),
+    fieldId: Type.String(),
+    value: Type.Any(),
```

Comment:
Nit: Type.Uknowen()  would be better, so we don't use it freely in the code without casting.